.github/workflows/swift-test.yml

```
@@ -31,18 +31,39 @@ jobs:
             xcode: "/Applications/Xcode_16.4.app"
             deviceName: "iPhone 16 Pro"
             osVersion: "18.5"
+
+          - type: ios
+            runs-on: macos-15
+            xcode: "/Applications/Xcode_26_beta.app"
+            deviceName: "iPhone 16 Pro"
+            osVersion: "26.0"
+            download-platform: "true"
   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)